tests/test_isrTaskLSST.py

```diff
@@ -99,6 +99,11 @@ def test_isrBootstrapBias(self):
         isr_config.doApplyGains = False
         isr_config.doBias = True
 
+        # Need to make sure we are not masking the negative variance
```

Comment:
The key here is that we don't want to mask negative pixels, because these are legal in calibration-corrected calibrations.  And `maskNegativeVariance` inadvertently does that.